lib/mintel/dashboards/services/omni/analytics.libsonnet

```diff
@@ -98,4 +98,49 @@ local promQuery = import 'components/prom_query.libsonnet';
       )
     ),
  ]),
+
+  dashboardRequest(span=12)::
+
+    layout.grid([
+
+      commonPanels.latencyTimeseries(
+        title='Dashboard Request Time',
+        description='Dashboard Request Time by dashboard id',
```

Comment:
Should we just use title case here

```
Dashboard Request Time by Dashboard ID
```

Reply:
I propose CamelCAse "DashboardRequestTimeByDashboardId"

:D 

fine

Reply:
Updated.